### DISCUSSION TITLE
Help verify "Reverse" or "[R]" in track names

### DISCUSSION THREAD

**Discussion by davidhcefx** [davidhcefx/Tracks-in-KartRider-Rush-Story-Mode#8] [Q&A] [answered]
Hello,

I need someone to help me verify that:

- Is it `[R] Up 'N' Down (Village)` or `Up 'N' Down Reverse (Village)`?
- And is it the same for `Cemetery Circuit Reverse (Tomb)` and `Canopy Curve Reverse (Forest)`?

@FauxBlue Can you help with this?
Thanks!

**Comment by FauxBlue** [comment, marked as answer]
All of the reverse tracks are prefixed with "[R] ", and are otherwise named the same as the original.

- [R] Jump Point (Northeu)
- [R] Up 'N' Down (Village)
- [R] Cemetery Circuit (Tomb)
- [R] Canopy Curve (Forest)
